### PR TITLE
Footnote citation spacing

### LINKED ISSUE #440: Footnotes have excessive margins
I don't know if I'm doing something wrong with my markup, but footnotes have very large spacing compared to their size.

```rst
6. **remote:** Remove old key from ``~/.ssh/authorized_keys``.
7. **client:** Delete old key files.

.. [1] https://wiki.mozilla.org/Security/Guidelines/OpenSSH#OpenSSH_client
.. [2] https://wiki.mozilla.org/Security/Guidelines/OpenSSH#OpenSSH_server
```

Produces

```html
  …
  <li><strong>remote:</strong> Remove old key from <code class="docutils literal"><span class="pre">~/.ssh/authorized_keys</span></code>.</li>
  <li><strong>client:</strong> Delete old key files.</li>
</ol>
<div class="wy-table-responsive">
  <table class="docutils footnote" frame="void" id="id3" rules="none">
  …
  </table>
</div>
<div class="wy-table-responsive">
  <table class="docutils footnote" frame="void" id="id4" rules="none">
  …
  </table>
</div>
```
Which under **sphinx-rtd-theme** looks like this:

footnotes

### PR DESCRIPTION
**Pull request by jessetan**
Old behaviour:
old

New behaviour:
new

This PR removes the `margin-bottom` from (the wrapper around) citations and footnotes, which makes them look better.
To preserve space between footnotes and whatever comes next, I've added a `margin-top` to any element that follows.

Fixes #440

### CODE REVIEW

File: sass/_theme_rst.sass [hunk 1]
```diff
@@ -174,6 +174,11 @@
       vertical-align: top
     code
       color: $gray
+  .wy-table-responsive.citation, .wy-table-responsive.footnote
```
Comment (Blendify) [inline sass/_theme_rst.sass]: Add a comment about what this is working around.

File: js/theme.js [hunk 2]
```diff
@@ -74,9 +74,15 @@ function ThemeNav () {
             })
 
         // Make tables responsive
```
Comment (Blendify) [inline js/theme.js]: Same here update the comment

File: sass/_theme_rst.sass (hunk 1 above)
Comment (jessetan) [reply sass/_theme_rst.sass]: Done (without mentioning GitHub issue)

File: js/theme.js (hunk 2 above)
Comment (jessetan) [reply js/theme.js]: Done (without mentioning GitHub issue)

**Comment by skirpichev** [comment]
@jessetan, I hope this also closes rtfd/sphinx_rtd_theme#479 (duplicate of #440?).

**Comment by jessetan** [comment]
@skirpichev I haven't built the entire diofant documentation, but if I paste the docstring into a demo document, it looks fine with this fix.

**Comment by skirpichev** [comment]
@jessetan, good news.  Thank you for your work.

BTW, I did some workarround [here](https://github.com/skirpichev/diofant/commit/cd3b6c715283555e24c0c9e34d4dc92bcb20fc31).  I don't see any bad side effects on my build, but perhaps there are.

**Pull request merged**